title_belt_nhl/static/nhl_tms.py

```diff
@@ -0,0 +1,34 @@
+nhl_team_abbvs = [
```

Comment:
If you make this an enum you could use it as the type for the team parameter elsewhere to restrict it more than a `str`. And you could use the names as the abbrevs and the values as the full names? Might not be worthwhile but just a thought.

```py
class NhlTeams(str, Enum):
  "NJD": "New Jersey Devils"
  "BUF": "Buffalo Sabres"
  # ..etc

# ... elsewhere
def func_that_takes_team(team: NhlTeams): ...
```